packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: 0.25;
```

Comment:
We shouldn't. Flexbox allocate the space left after the elements were assigned a size. So if an element has no size, it will keep its original one, and elements with flex will grow to fill up the remaining space. 
In this case, we can either have a fixed width (which I don't love because the text might change, user might zoom in page, ...), or either having no width nor flex, only a simple padding so it doesn't look too small.